<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_promatposrumpf</class>
 <widget class="QDialog" name="Dialog_promatposrumpf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_new">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>neu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>entf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xxx Artike:l</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_filter">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_favourite">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>256</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_usefavourite">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>verwenden</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_favourite">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Favouriten:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>311</width>
     <height>351</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_filter</tabstop>
  <tabstop>pushButton_new</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>pushButton_edit</tabstop>
  <tabstop>listWidget_favourite</tabstop>
  <tabstop>pushButton_usefavourite</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
